packages/fxa-settings/src/components/UnitRow/index.stories.tsx

```diff
@@ -91,4 +93,40 @@ storiesOf('Components|UnitRow', module)
         )}
       </UnitRow>
     );
+  })
+  .add('with default avatar', () => {
+    const avatar = {
+      id: null,
+      url: null,
+      isDefault: true,
+    };
+    return (
+      <MockedCache account={{ avatar }}>
+        <UnitRow
+          header="Picture"
+          headerId="profile-picture"
+          headerValue={!avatar.isDefault}
+          route={`${HomePath}/avatar`}
+          {...{ avatar }}
+        />
+      </MockedCache>
+    );
+  })
+  .add('with non-default avatar', () => {
+    const avatar = {
+      id: null,
+      url: 'http://localhost:1111/v1/avatar/t',
```

Comment:
Since we push Storybook builds up to this URL, it'd be more ideal to change this to something that'll work everywhere. We tend to use silly placekitten images, if you change this to `http://placekitten.com/512/512?image=0` we'll be golden. 😛